presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -104,6 +104,15 @@ public static long charLength(@LiteralParameter("x") long x, @SqlType("char(x)")
         return x;
     }
 
+    @Description("count of bits for the given string")
+    @ScalarFunction("bit_length")
+    @LiteralParameters("x")
```

Comment:
This literal parameter is not needed?

Reply:
Here I refer to the implementation of the length function in StringFunctions.java.